### ISSUE TITLE
Disable Liquidity Rewards

### ISSUE THREAD

**Issue by mvandeberg** [steemit/steem#178]
Due to the controversy of liquidity rewards we are going to evaluate our design and change liquidity rewards if needed. For now, we believe they are not providing enough value to keep around. We have decided to disable liquidity rewards altogether for the time being while we work on a more permanent solution. This is a temporary change. The internal market not charging transaction fees may be enough of an incentive to trade internally that we may not need liquidity rewards at all. All of these aspects will go into our final decision. We encourage the community to voice their thoughts on liquidity rewards and potential changes in a civil and objective manner.

**Comment by iamsmooth** [comment]
With conditions continuing to evolve rapidly after the last set of rule changes and continued growth in the number of traders in the market, maybe some longer period of evaluation is called for before taking radical action. I don't have a strong view either way though.

**Comment by mvandeberg** [comment]
This hardfork is not scheduled to go live until July 26th, 2016 at 15:00:00 UTC. We are currently in process of reviewing all of the code changes and fixing problems as we find them. If there are strong opinions about the changes on either side, please voice them. At the end of the day, it is up to the witnesses to update their node and vote for a hardfork.

**Comment by iamsmooth** [comment]
There will be a lot of changes in the hard fork I suspect, making 'take it or leave it' a less useful approach than actual consideration on a specific change. As I said I don't have strong opinion here but I just think more than a couple of days of of evaluation are appropriate before reaching a conclusion on whether to include this. My opinion anyway.

**Comment by jl777** [comment]
1 minute was too fast, 30 minutes too slow. 30x increase was quite drastic.
Maybe a 5 minute requirement will be enough to create risk for the self-traders, but my bot that doesnt self-trade is getting small fraction of the points that the self-trading bots do.

I know sybil accounts make it near impossible to prevent effective selftrading, but if none of the participants selftraded, I think 5 minute time would work out well. To prevent self-trading some sort of analogue to coinage might be needed. that would avoid the need for path analysis, which can be gamed anyway.

Another approach is to just limit the size of bid/ask that is counted toward the liquidity award. Having 10000 STEEM orders selftrading is really just silly and only being done for the points.

Another approach would be to have a small number of knowledgeable voters to assign percentages of points they should get based on qualitative factors, but of course this is quite messy. 

Now I have a halfway decent bot done, I would keep it running as long as I am able to get some sort of subsidy. without any subsidy what would happen is that one day the server would be restarted for maintenance and I just wouldnt find the time to start it up again as the volatility risk does outweigh the gains from spreads from the real trading. As it is the other bots (humans?) tried to fool my bot into lower and lower prices in hopes to drain it of capital but I am tracking implied values from external markets and constraining trades based on that. Though I try to push the price incrementally closer to the theoretical value as that creates the least stress and least risk.

tl:dr 5 minutes and a cap of 100 STEEM size per order is probably worth trying, or have some human involved way to award subsidies. The idea of having a fully computerized metric to award liquidity providers is quite courageous, not sure if it has ever been done before
